apps/framework-internal-app/package.json

```diff
@@ -3,10 +3,10 @@
   "version": "0.0",
   "scripts": {
     "dev": "moose-cli dev",
-    "build": "moose-cli build --docker"
+    "build": "moose-cli --version && moose-cli build --docker"
   },
   "dependencies": {},
   "devDependencies": {
-    "@514labs/moose-cli": "0.3.199"
```

Comment:
This is not going to bump the version unless you do a `pnpm install` the lock file has not changed in this PR.

That's why I set the version explicitly here is to emphasize that this needs to go through a regular update process to get updated